chia/full_node/mempool.py

```diff
@@ -101,7 +105,13 @@ def _row_to_item(self, row: sqlite3.Row) -> MempoolItem:
         item = self._items[name]
 
         return MempoolItem(
-            item.spend_bundle, uint64(fee), item.npc_result, name, uint32(item.height_added_to_mempool), assert_height
+            item.spend_bundle,
+            uint64(fee),
+            item.npc_result,
+            name,
+            uint32(item.height_added_to_mempool),
+            assert_height,
+            dedup_spend_info=item.dedup_spend_info,
```

Comment:
it's a bug not to set the `assert_before_*` fields. I'll file a separate PR to fix that